### PR TITLE
FIX: missing debug modes

### PR DESCRIPTION
**Pull request by haslinghuis**

### CODE REVIEW

**Review by nerdCopter — APPROVED** [review]
* works.
* wondering if debugs for angle should be renamed for sorting purposes? e.g. `angle_current` `angle_rate` and maybe others e.g. gyro_dual_* (instead of dual_gyro_*)

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by haslinghuis** [comment]
Yes as having proposed in the firmware PR

**Review by McGiverGim — APPROVED** [review]

**Pull request merged**
